metaseq/service/utils.py

```diff
@@ -32,6 +33,14 @@ def encode_fn(generator, x):
     return generator.bpe.bpe.encode(normalize_newlines(x)).ids
 
 
+def decode_fn(generator, x: List[int]) -> str:
```

Comment:
dumb q - where is this being used?

Reply:
It will be used for evals internally... I could have added it there, but adding it here to keep related functions in the same place...

Reply:
maybe it would be better to just move this to `generator` itself? Same with encode_fn

Reply:
Moved these functions to the generator